mfr/extensions/tabular/libs/h5py_scipy_tools.py

```diff
@@ -0,0 +1,60 @@
+import h5py
+import scipy.io
+
+from ..utilities import data_population, header_population
+from collections import OrderedDict
+
+
+def mat_v73(fp):
+    """Read and convert a mat v7.3+ file to JSON format using h5py
+    :param fp: File pointer object
+    :return: tuple of table headers and data
+    """
+
+    workbook = h5py.File(fp.name, 'r')
+    sheets = OrderedDict()
+    variables = workbook.items()
+
+    for var in variables:
+        name = var[0]
+        data = var[1]
+        if type(data) is h5py.Dataset:
+            #h5py seems to swap cols/rows for some reason. this fixes it so it displays like it does in matlab
+            list_data = list(zip(*data.value.tolist()))
+            #add a default header to fix some data formats
```

Comment:
Can we make this a helper method since it is used by both `v73` and `v7`?

Reply:
And it will be great if you can add a comment/explanation on the `list_data` format. Not a blocker.